CHANGELOG.md

```diff
@@ -39,6 +39,10 @@ For older changes see the [archived Singularity change log](https://github.com/a
   pkg/build/types/parser.All(), `.Raw` contains the raw content of a single
   build stage. Otherwise, it is equal to `.FullRaw`.
 
+### Bug fixes
+
+- Fix the build failure when unprivileged user namespaces is unavailable.
```

Comment:
This can go in the next Bug fixes section instead of adding a new one.  Change `when unprivileged` to `when in setuid mode and unprivileged`.  Change `is` to `sre`.

Reply:
updated in this commit
https://github.com/apptainer/apptainer/pull/1586/commits/7357e451fa67608510c8a6ea50f13fca49163d61